app/socket-apis/__tests__/upsert-rank.js

```diff
@@ -0,0 +1,103 @@
+import rankWhy from '../rank-why'
```

Comment:
When I look  at this code in visual studio code I see that rankWhy isn't being used (it's gray).  So please remove it.